app/src/main/java/io/capibaras/abcall/viewmodels/SignUpViewModel.kt

```
@@ -0,0 +1,94 @@
+package io.capibaras.abcall.viewmodels
+
+import android.content.Context
+import android.util.Patterns
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import io.capibaras.abcall.R
+
+class SignUpViewModel() : ViewModel() {
```

detekt

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)